specs/cli/show.md

```diff
@@ -0,0 +1,36 @@
+# show
+
+## Description 
+
+Show the API paths tree for an API description.
+
+## Parameters
+
+| Parameters | Required | Example | Description | Telemetry |
+| -- | -- | -- | -- | -- |
+| `--openapi \| -d` | Yes | https://aka.ms/graph/v1.0/openapi.yaml | The location of the OpenAPI description in JSON or YAML format to use to generate the SDK. Accepts a URL or a local path. | Yes, without its value |
+| `--clear-cache \| --cc` | No | true | Clears any cached data for the current command. Defaults to `False`. | Yes |
+| `--log-level \| --ll` | No | Critical | The log level to use when logging messages to the main output. Options available: Critical, Debug, Error, Information, None, Trace & Warning. Defaults to `Warning`. | Yes |
+| `--version \| --v` | No | beta | The version of the OpenAPI document to use | Yes, without its value |
+| `--search-key \| --k` | No | github::microsoftgraph/msgraph-metadata/graph.microsoft.com/v1.0 | The search key used to locate the OpenAPI description. | Yes, without its value |
+| `--max-depth \| --m-d` | No | 10 | The maximum depth of the tree to display. Defaults to `5`. | Yes, without its value |
```

Comment:
Should we provide a "all" value (or something else) so people can get all results no matter the depth?

Reply:
We decided to not as it could lead to endless loops.

Reply:
We decided to not as it could lead to endless loops.